frontend/web/components/modals/Rule.js

```diff
@@ -65,7 +65,7 @@ export default class Rule extends PureComponent {
                                   data-test={`${this.props['data-test']}-value-${i}`}
                                   className="input-container--flat full-width"
                                   value={`${value}`}
-                                  placeholder="Value *"
+                                  placeholder={operator && operator === 'MODULO' ? 'Value(<divisor>|<remainder>) *' : 'Value *'}
```

Comment:
I was hoping that we'd be able to split the form into 2 separate inputs if the operator was MODULO... but I am by no means a react developer so I'm not sure how challenging this would be. What do you think @kyle-ssg ?

Reply:
@nogoodusername I love this idea but I agree here, is it possible to edit the Rule component such that there would be 2 inputs for this rule type?

Reply:
Initially, I was thinking the same but I think we'd want to do this in a way so that future rules could have multiple inputs too. With that being said I think the placeholder approach is fine for now